skfp/descriptors/topological.py

```diff
@@ -288,6 +325,129 @@ def polarity_number(
     return int((distance_matrix == 3).sum() // 2)
 
 
+def kappa1_index(mol: Mol) -> float:
+    r"""
+    First Kappa shape index (K1).
+
+    Computes the first kappa shape index [1]_, which measures molecular shape based on
+    single bonds. It is given by the equation:
+
+    .. math::
+        K_1 = \frac{(A + \alpha) (A + \alpha - 1)^2}{P_1^2}
+
+    where:
+    - A is the number of heavy atoms,
+    - α is the Hall-Kier alpha index,
+    - P1 is the number of single bonds.
```

Comment:
We do not add commas or dots with lists. Also note that you must have a single empty line between `where:` and first list element, otherwise Sphinx will not render docs properly